formats/role_appointment.jsonnet

```diff
@@ -0,0 +1,23 @@
+(import "shared/default_format.jsonnet") + {
+  base_path: "forbidden",
+  routes: "forbidden",
+  definitions: {
```

Comment:
Does this need a `role` and `person` edition link type?

Reply:
Indeed, added